### PR TITLE
repo rename

### PR DESCRIPTION
**Pull request by tjwebb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +16,7 @@ deploy:
...
+    secure: UJKl70RkuFLJWOJViDkVklykAVYZmiPjlLku2H0VgJqOnnKDr6HW1sOvp7mMouI89WsKY9Mh0j1UHPS5zz5AxsMx22omYExrTlxDhLg7xUMXFh3m3oX7ZiiasKuDtjoXH7d/SwMWThr+c25KmoPnmVUpOcdyKZmY051fAK/AxwOnEZ+XX/v5TCRwrUzuZ+9lOZ3E9FA3Athc7sE9MvVErB+MFXKpgNUSlahhLeYvFhpsNOes38ictVpNY/cF59lzTD3KFx0lC6UamP1b2fgVB/tdm0s8Ws/bxo4MOKb8Q4LiUJbKYuZzU/nsImFC+8MRkcTDS7/qqkd64j0sAq4L5Sei+9z/mIlT7iUBEYma3iU+Zc0Gqn90lwLMloI1oSp6Mf/L+8M8BpQLgT1aNA3cTmUhOg7a9MWEQMajQKAcp975wk0/ztJfVE7PRBrVGmbhZuEN3dpjpQqxSCNyYUNHvydgwPKCUKN06qli8fk80+P7PK+e8op13K/kLIejc2jn/8tXgXsJNNRWCiYu3aMLUy6mNgjFa7Dvm0+xeQ/tMBD85U2eYo61UmJi7aTzEYcR/QsLgFPHE9Ng1h9O/+K/IrbRVa6Q1QY4JJje8CXZzSPBDbVl+PPvAiyxmmnxP09rSvkoAgjbODUzi9XdxuscpCiTq1shr85roNfquBKK1GA=
   on:
     tags: true
     repo: trailsjs/trailpack-express
-    node: stable
```
Comment (jaumard) [inline .travis.yml]: If you remove when the publish will occur ?

**Comment by jaumard** [comment]
On this PR (https://github.com/trailsjs/trailpack-express/pull/17) I move express into devDep (for tests)

File: .travis.yml (hunk 1 above)
Comment (jaumard) [inline .travis.yml]: Up @tjwebb :)

Comment (tjwebb) [inline .travis.yml]: On tags, per usual.

Comment (jaumard) [inline .travis.yml]: but if you don't specify a node version it will try to publish 3 times ? 4 5 and 6 ?

Comment (tjwebb) [inline .travis.yml]: Sorry, I thought I replaced it with node 6. I'll fix.

**Pull request merged**